x-pack/plugins/security/server/authentication/providers/saml.test.ts

```diff
@@ -231,6 +231,131 @@ describe('SAMLAuthenticationProvider', () => {
       );
     });
 
```

Comment:
nit: I think we're missing a test to verify that an SP initiated login ignores `RelayState` if provided (and `useRelayStateDeepLink` is `true`)